en/resources/template-engines.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Template Engines
+menu: resources
+lang: en
+redirect_from: "/resources/template-engines.html"
+---
+
+# Template engines out-of-the-box compatible with Express
```

Comment:
Rather than provide details in the heading, move to text:

# Template engines

The following template engines are compatible "out-of-the-box" with Express: